### PR TITLE
Variables

### PR DESCRIPTION
**Pull request by loverajoel**
## What does this Pull Request Do?
That's my proposal.
Use one ```var``` declaration per variable. It's easier to add new variable declarations this way, and you never have to worry about swapping out a ```;``` for a ```,``` or introducing punctuation-only diffs. Use one ```var``` declaration for multiple variables and declare each variable on a newline only when the vars have relationship.
```javascript
var items = getItems(),
    itemsCant = getItems().length;
var dragonball = 'z';
```

### CODE REVIEW

**Comment by charly22** [comment]
:+1:

**Comment by maxigimenez** [comment]
:shipit:

File: README.md [hunk 1]
```diff
@@ -363,7 +363,7 @@
     var superPower = new SuperPower();
     ```
 
-  - Use one `var` declaration for multiple variables and declare each variable on a newline.
+  - Use one ```var``` declaration per variable. It's easier to add new variable declarations this way, and you never have to worry about swapping out a ```;``` for a ```,``` or introducing punctuation-only diffs. Use one ```var``` declaration for multiple variables and declare each variable on a newline only when the vars have relationship.
```
Comment (homer0) [inline README.md]: Let me know what you think:

> User one `var` statement per variable declaration. It's easier to read and prevent possible errors that are difficult to catch when you use one `var` for multiple declarations. There's only one special case when this is allowed and it depends on the programmer: If the declarations are related and/or you feel they should be grouped, you can use one `var` for multiple declarations, see the examples below to get a better understanding of this:

**Comment by homer0** [comment]
Other than my comment, it's a :shipit:

**Pull request merged**